### ISSUE TITLE
Issue with escaping backslash

### ISSUE THREAD

**Issue by saadq** [gandm/language-babel#337]
First off, thanks a lot for this great package!

There seems to currently be an issue when trying to escape a backslash (`\`) at the very end of the template literal. Here is an extremely minimal example that can be reproduced:

```javascript
const str = `Hello \\`
const str2 = `Hey \\`

console.log(str) // => Hello \
console.log(str2) // => Hey \
```

This results in the syntax highlighting getting corrupted:

![screen shot 2017-03-08 at 1 14 43 pm](https://cloud.githubusercontent.com/assets/5678694/23717327/5f27bac6-0401-11e7-92c5-63efeece1397.png)

I believe the issue is that it seems to think that the closing ` backtick is supposed to be escaped. If I add a space after the last backslash, the syntax highlighting gets fixed:

![screen shot 2017-03-08 at 1 17 04 pm](https://cloud.githubusercontent.com/assets/5678694/23717372/8a8c9cc2-0401-11e7-8b9e-3a2e13053269.png)

Thanks for your time.

**Comment by gandm** [comment]
Thanks for the report. I can see the issue. The \\ is being matched OK but the close backtick is being ignored because I have a regex that ignores it if preceded by a \. I'll fix soon.

**Comment by gandm** [comment]
Should be fixed in 2.56.1

**Comment by saadq** [comment]
Thanks a lot for being so quick! Much appreciated.
